### PR TITLE
[BOAT] Timestamps in deleted log

### PR DESCRIPTION
**Pull request by BenSegal855**
This PR adds timestamps of deleted messages in the deleted log. It also adds the duration between when the message was created and when it was deleted.
- note: Adds pretty-ms as a node dependency

### CODE REVIEW

File: boat/logger.js [hunk 1]
```diff
@@ -40,11 +42,14 @@ module.exports = {
...
       const cleanMessage = msg.cleanContent.replace(/`/g, `\`${zws}`)
       const cleanUsername = msg.author.username.replace(/@/g, `@${zws}`).replace(/`/g, `\`${zws}`)
+      const time = new Date(msg.timestamp)
       bot.createMessage(config.discord.ids.channelMessageLogs, template
         .replace('$channelId', msg.channel.id)
         .replace('$username', cleanUsername)
         .replace('$discrim', msg.author.discriminator)
         .replace(/\$userId/g, msg.author.id)
+        .replace('$time', time.toUTCString())
+        .replace('$duration', prettyMs(Date.now() - time))
```
Comment (cyyynthia) [inline boat/logger.js]: Is pretty-ms really required? The date could be broken down to a duration easily with a relatively simple function

Comment (BenSegal855) [reply boat/logger.js]: Resolved by [d1101ef](https://github.com/powercord-org/powercord-backend/pull/39/commits/d1101ef24fbfd192ec58b63694b84e2440d47358)

Comment (BenSegal855) [reply boat/logger.js]: nope nevermind, I made a dumb. fix in soon

Comment (BenSegal855) [reply boat/logger.js]: Dumb fixed by [7ef983b](https://github.com/powercord-org/powercord-backend/pull/39/commits/7ef983bbae274f8eed60ee8edfa52025fb4d820f)

**Review by cyyynthia — APPROVED** [review]

**Pull request merged**